### DISCUSSION TITLE
Silent install ligger sig under bruger i appdata i stedet for alle brugere

### DISCUSSION THREAD

**Discussion by trueTLW** [Eduap-com/WordMat#176] [Support]
### Discussed in https://github.com/Eduap-com/WordMat/discussions/172

<div type='discussions-op-text'>

<sup>Originally posted by **trueTLW** January 11, 2024</sup>
Jeg har installeret tidligere versioner med disse parametre men nu ligger den sig kun for den bruger som er logget på.
/COMPONENTS=GRAPH /SUPPRESSMSGBOXES /SILENT /VERYSILENT /SP")
Hvis jeg manuelt installere det og tilvælger "alle brugere" så virker det fint for alle som logger på.
Sætter jeg "/ALLUSERS" så vælger den stadig kun "lokal bruger".
Det er både 1.28 og 1.28.1 som giver det problem.</div>

**Comment by Eduap-com** [comment]
Kan du melde tilbage om følgende parameter virker:
/TASKS=”installeralle”

**Comment by trueTLW** [comment]
Det hjalp på det det. Disse parameter installere det silent for alle brugere: (Graph bliver også lagt på). Tak for svaret.

/SP /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /COMPONENTS=GRAPH /TASKS=installeralle
